modules/secure-serverless-net/variables.tf

```diff
@@ -77,3 +77,8 @@ variable "resource_names_suffix" {
   type        = string
   default     = null
 }
+
+variable "serverless_type" {
+  description = "The type of resource to be used. It supports only CLOUD_RUN or CLOUD_FUNCTION"
+  type        = string
+}
```

Comment:
Can we add [validation](https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules) if we only accept 2 distinct values?

something like: 
```
validation {
    condition     = contains(["CLOUD_RUN", "CLOUD_FUNCTION"], var.serverless_type)
    error_message = "unsupported value for serverless_type"
  }
```
and elsewhere where we are creating logic based on this variable